Services/coffee_shop_service.py

```diff
@@ -0,0 +1,27 @@
+from Models.coordinates import Coordinates
+from Models.coffee_shop import CoffeeShop
+
+
+class CoffeeShopService:
+
+    def generate_coffee_shops(self, csv_data):
+        coffee_shops = []
+        for row in csv_data:
+            try:
+                shop_name = row[0]
+                shop_coordinates = Coordinates(row[1], row[2])
+                shop = CoffeeShop(shop_name, shop_coordinates)
+                coffee_shops.append(shop)
+            except:
+                print(f"Invalid data ... Skipping over row {row} ")
```

Comment:
Is great that you try to parse as many results as possible,

The problem is that you are mixing up presentation with business logic. If you service is to be used from a web app for example the print here won't be very useful to the end user.